docs/cli.rst

```diff
@@ -1,11 +1,271 @@
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+        when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Troubleshooting Errors
+----------------------
+
+Errors happen! That's why our team strives to provide our users with options for efficiently solving them when they occur.
+
+
+Reporting Error Logs 
+^^^^^^^^^^^^^^^^^^^^
+
+The ``cci error gist`` command sends the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. For this feature to work you need to ensure that your `github service is set up with the proper scopes <https://cumulusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
+
+Information included in the gist:
+
+* The current version of ``cci``
+* The current Python version
+* The path to the Python executable
+* The ``sysname`` of the host (such as Darwin)
+* The machine name of the host (such as x86_64)
+* The most recent log file (cci.log) that CumulusCI has created.
+
+The URL for the gist is displayed on the user terminal as output, and a web browser automatically opens a tab to the gist.
+
+
+The ``--debug`` Option
+^^^^^^^^^^^^^^^^^^^^^^
+
+All CumulusCI commands can be passed the ``--debug`` option. When the option is used:
+
+* Any calls to CumulusCI's logger at the debug level are shown.
+* Outgoing HTTP requests are logged.
+* If an error is present, the corresponding stacktrace is shown, and the user is dropped into a `post-mortem debugging <https://docs.python.org/3/library/pdb.html#pdb.post_mortem>`_ session.
+
+.. note:: To exit a debugging session type the command ``quit`` or ``exit``.
+
+
+Log Files
+^^^^^^^^^
+
+CumulusCI creates a log file every time a cci command runs. There are six rotating log files (``cci.log, cci.log1...5``) with ``cci.log`` being the most recent. Log files are stored under ``~/.cumulusci/logs``.
+
+By default, log files document:
+
+* The last command that was entered by the user.
+* All output from the command (including debug information).
+* If a Python-level exception occurs, the corresponding stacktrace is included.
```

Comment:
This bullet item isn't parallel with the others. I think you can just omit "is included" to make it logically complete the "By default" stem phrase.